pool/src/types.rs

```diff
@@ -107,6 +107,11 @@ pub struct PoolConfig {
 	#[serde(default = "default_accept_fee_base")]
 	pub accept_fee_base: u64,
 
+	// Reorg cache retention period in minute.
+	// The reorg cache repopulates local mempool in a reorg scenario.
+	#[serde(default = "default_reorg_cache_period")]
+	pub reorg_cache_period: i64,
```

Comment:
Lets make this `u64` as it does not make sense to support `-ve` values in the config.

Reply:
I can't do `u64`, as [minutes()](https://docs.rs/chrono/0.4.11/chrono/struct.Duration.html#method.minutes) requires `i64`. Either we set `u32` and turn it to `i64` in minutes() or we just let `i64` but no error will be thrown if users put a negative value.

Reply:
`u32` works then, we're not going to need huge values in here.